examples/libtorchaudio/speech_recognition/README.md

```diff
@@ -2,10 +2,6 @@
 
 This example demonstarates how you can use torchaudio's I/O features and models to run speech recognition in C++ application.
 
-**NOTE**
-This example uses `"sox_io"` backend for loading audio, which does not work on Windows. To make it work on
-Windows, you need to replace the part of loading audio and converting it to Tensor object.
-
```

Comment:
Let's put back this file as it originally was, and create a follow-up PR for all the `example/` at once. Let's add a notice the README of all the `examples/` that aren't tutorials and indicate that a bunch of those are probably relying on removed APIs and that these example only exist for reference but may not work as-is.